modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -24,33 +19,80 @@ Deleting a persistent volume that is still in use can result in data loss or cor
 
 .Procedure
 
-. Edit the previously created local volume to remove any unwanted disks.
+To delete LVs or LVSs, complete the following steps:
+
+. If there are any bound PVs, delete the corresponding persistent volume claims (PVCs) to release the PVs:
 
-.. Edit the cluster resource:
+.. To find bound PVs, run the following command:
 +
+[source, terminal]
+----
+$ oc get pv
+----
++
+.Example output
 [source,terminal]
 ----
-$ oc edit localvolume <name> -n openshift-local-storage
+NAME   CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS      CLAIM                 STORAGECLASS   VOLUMEATTRIBUTESCLASS   REASON   AGE
+pv1    5Gi        RWO            Retain           Available                         slow           <unset>                          28s
+pv2    30Gi       RWX            Retain           Bound       openshift/storage     my-sc          <unset>                          168d
```

Comment:
I would suggest using PV names like `local-pv-1cec77cf` in this example to stick with the naming scheme LSO uses when creating them.